packages/appChrome/types.ts

```diff
@@ -1,4 +1,3 @@
-import { Theme } from "../themes/types/appTheme";
```

Comment:
was this used anywhere else? Shoudl the file be deleted?

Reply:
Imports elsewhere will now be `import { LocalTheme as Theme } from "../types/appTheme";` with AppChromeTheme no longer extending Theme here it could be removed.